microsoft-edge/webview2/how-to/webdriver.md

```diff
@@ -103,24 +103,20 @@ At this point you should have WebView2 Runtime installed, built a WebView2 proje
     | `is_legacy` | Set to `false`, which tells Selenium that you are driving the new Chromium-based Microsoft Edge browser. |  
```

Comment:
Please remove this table. The EdgeOptions object no longer uses constructor parameters. The UseWebView and UseChromium properties have replaced these.

Reply:
done!